### PR TITLE
A logged-in member can view their dues amount.

### PR DESCRIPTION
**Pull request by schlich**

### CODE REVIEW

File: dsa_actionkit/contexts/account_contexts.py [hunk 1]
```diff
@@ -1,318 +1,320 @@
...
     list_ids = [1, 3, 7]
 
 
 contexts = {
-    'user_view.html': {
-        'filename': 'user_view.html',
-        'page': {
-        'title': 'User self serve',
-        'name': 'user view',
+    "user_view.html": {
```
Comment (swoopej) [inline dsa_actionkit/contexts/account_contexts.py:11]: are you using a formatter for these is it in a pre-commit hook or something? if not can we put a little about it in the readme so we don't have reformat stuff a bunch?

(I feel like someone mentioned adding precommit hooks in one of the meetings but now I forget sorry)

File: pyproject.toml [hunk 2]
```diff
@@ -39,3 +41,18 @@ test = ["selenium>=3.8.0", "pyvirtualdisplay"]
...
+fix = true
+select = ["ALL"]
+per-file-ignores = { "tests/**/*.py" = [
+    "S101",
+    "D100",
+    "D103",
+    "ANN201",
+    "ANN001",
+    "INP001",
+] }
```
Comment (swoopej) [inline pyproject.toml:55]: oh this looks like the linter stuff got it

File: tests/test_user_landing_page.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import pytest
+from django.shortcuts import reverse
+
+
+@pytest.mark.parametrize("dues_amount", (3, 100))
```
Comment (swoopej) [inline tests/test_user_landing_page.py:5]: parametrize -> parameterize?

**Comment by swoopej** [comment]
love the refactor, already helps me follow things a bit better I think

File: dsa_actionkit/contexts/account_contexts.py (hunk 1 above)
Comment (schlich) [reply dsa_actionkit/contexts/account_contexts.py:11]: Used reference in new issue to follow up on this [here](https://github.com/dsa-ntc/actionkit-templates/issues/12): 

image

Please feel free to use this feature liberally :)

File: tests/test_user_landing_page.py (hunk 3 above)
Comment (schlich) [reply tests/test_user_landing_page.py:5]: yep -- rushed this one out and missed that.  Can you try step number 6 on [this page](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) to try and make an inline edit with this change?

**Pull request merged**